s3fs/core.py

```diff
@@ -71,6 +71,59 @@ def split_path(path):
              'authenticated-read'}
 
 
+def title_case(string):
+    """
+    TitleCases a given string.
+
+    Parameters
+    ----------
+    string : underscore seperated string
+    """
+    return ''.join([x.capitalize() for x in string.split('_')])
+
+
+class SSEParams(object):
```

Comment:
So this is not really used, since we can provide arbitrary writer_kwargs - do you think it's necessary now? The only thing it really does, is list the fields we expect to be required specifically for SSE, which to me seems more like a documentation task.